src/main/kotlin/skytils/skytilsmod/features/impl/mining/MiningFeatures.kt

```diff
@@ -369,11 +369,12 @@ class MiningFeatures {
         enum class Locations(val displayName: String, val id: String, val color: Int, val size: Int = 50) {
             LostPrecursorCity("§fLost Precursor City", "internal_city", ColorFactory.WHITE.rgb),
             JungleTemple("§aJungle Temple", "internal_temple", ColorFactory.GREEN.rgb),
-            GoblinQueensDen("§eGoblin Queens Den", "internal_den", ColorFactory.YELLOW.rgb),
+            GoblinQueensDen("§eGoblin Queen's Den", "internal_den", ColorFactory.YELLOW.rgb),
```

Comment:
I think this still requires a change to https://github.com/Skytils/SkytilsMod/blob/e098fcc50a6f2999e372e7c39c7213c2aed320cf/src/main/kotlin/skytils/skytilsmod/utils/SBInfo.kt#L65

Reply:
I think it is fine? Unless you see something that I am missing. It works ingame and using a Regex checker, it detects "Goblin Queen's Den" fine.